Configuration/Geometry/python/GeometryExtended2023TkOnlysimReco_cff.py

```diff
@@ -0,0 +1,51 @@
+import FWCore.ParameterSet.Config as cms
+
+# Ideal geometry, needed for transient ECAL alignement
+from Configuration.Geometry.GeometryExtended2023TkOnlysim_cff import *
```

Comment:
@sviret - IMHO, it would be nice to have consistent capitalization in new file names (e.g. geometry scenarios): TkOnlySim and TkOnlyTilted.
BTW, Shouldn't TkOnlySim be TkOnlyFlat (as contrary to Tilted, rather then contrary to Reco)?